tools/testsys/src/run.rs

```diff
@@ -65,6 +66,10 @@ pub(crate) struct Run {
     #[clap(long, env = "BUILDSYS_OVA")]
     ova_name: Option<String>,
 
+    /// The name of the image that should be used for Bare Metal testing
+    #[clap(long, env = "BUILDSYS_NAME_FULL")]
```

Comment:
Does this include the extension? i.e. is this the file name?

Reply:
This is just a cargo make env variable that we are re-using.